nilearn/datasets.py

```diff
@@ -1039,17 +1044,34 @@ def fetch_smith_2009(data_dir=None, url=None, resume=True, verbose=1):
     http://www.fmrib.ox.ac.uk/analysis/brainmap+rsns/
     """
     if url is None:
-        url = "http://www.fmrib.ox.ac.uk/analysis/brainmap+rsns/"
-
-    files = [('rsn20.nii.gz', url + 'rsn20.nii.gz', {}),
-             ('PNAS_Smith09_rsn10.nii.gz',
-                 url + 'PNAS_Smith09_rsn10.nii.gz', {}),
-             ('rsn70.nii.gz', url + 'rsn70.nii.gz', {}),
-             ('bm20.nii.gz', url + 'bm20.nii.gz', {}),
-             ('PNAS_Smith09_bm10.nii.gz',
-                 url + 'PNAS_Smith09_bm10.nii.gz', {}),
-             ('bm70.nii.gz', url + 'bm70.nii.gz', {}),
-             ]
+        if mirror == 'origin':
+            url = "http://www.fmrib.ox.ac.uk/analysis/brainmap+rsns/"
+        elif mirror == 'nitrc':
+            url = [
+                    'https://www.nitrc.org/frs/download.php/7730/',
+                    'https://www.nitrc.org/frs/download.php/7729/',
+                    'https://www.nitrc.org/frs/download.php/7731/',
+                    'https://www.nitrc.org/frs/download.php/7726/',
+                    'https://www.nitrc.org/frs/download.php/7728/',
+                    'https://www.nitrc.org/frs/download.php/7727/',
+            ]
+        else:
+            raise ValueError('Unknown mirror "%s". Mirror must be "origin" '
+                'or "nitrc"' % str(mirror))
+
+    files = [
+            'rsn20.nii.gz',
+            'PNAS_Smith09_rsn10.nii.gz',
+            'rsn70.nii.gz',
+            'bm20.nii.gz',
+            'PNAS_Smith09_bm10.nii.gz',
+            'bm70.nii.gz'
+    ]
+
+    if isinstance(url, _basestring):
```

Comment:
would it not be more readable to have this inside the `if mirror == 'origin'` clause ?